packages/shared/ReactError.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+// Do not require this module directly! Use a normal error constructor with
+// template literal strings. The messages will be converted to ReactError during
+// build, and in production they will be minified.
+
+function ReactError(message) {
+  const error = new Error(message);
+  error.name = 'Invariant Violation';
```

Comment:
While we're at it, can we just say "React Error"? Is that a breaking change? 😄

Reply:
I'd be comfortable changing in a minor

Reply:
I think it's a breaking change. It breaks continuity in logs.